src/utils/currency/config.ts

```diff
@@ -0,0 +1,30 @@
+import { Network, SupportedCurrencies } from '~gql';
+
+// For walkthrough, see: https://apiguide.coingecko.com/getting-started/10-min-tutorial-guide/1-get-data-by-id-or-address
+export const currencyApiConfig = {
+  endpoint: 'https://api.coingecko.com/api/v3/simple/token_price/',
```

Comment:
Initial reaction would be to have this as an `.env` var, but then again, this shouldn't change that often (or based on the environment) to warrant that